11-lorawan/test_spec11.py

```diff
@@ -0,0 +1,118 @@
+import time
+import pytest
+from testutils.shell import GNRCLoRaWANSend, ifconfig, lorawan_netif
+from riotctrl_shell.netif import Ifconfig
+import base64
+import json
+
+APP = 'examples/gnrc_lorawan'
+pytestmark = pytest.mark.rc_only()
+
+APP_PAYLOAD = "This is RIOT!"
+DOWNLINK_PAYLOAD = "VGhpcyBpcyBSSU9U"
+APP_PORT = 2
+RX2_DR = 3
+
+
+class Shell(Ifconfig, GNRCLoRaWANSend):
+    pass
+
+
+def run_lw_test(node, client, userdata, iface, app_id, dev_id):
```

Comment:
Requiring only

```suggestion
def run_lw_test(node, client, iface, app_id, dev_id):
```

for this function